### DISCUSSION TITLE
Ban events

### DISCUSSION THREAD

**Discussion by Alix-Menard** [TwitchIO/TwitchIO#378] [Questions]
I'm not used to handle events and I don't know how to make my bot wait for a banning event and use the datas afterwards, could someone help me please ?

**Comment by chillymosh** [comment]
Hi Alix, you will need to use the eventsub extension for this https://twitchio.dev/en/latest/exts/eventsub.html
Be aware that this does currently require a publicly accessible URL, non self-signed SSL cert and a reverse proxy, such as NGINX.

You will need to swap change the below for the appropriate events you wish to subscribe to
```
esclient.subscribe_channel_follows(broadcaster=channel_ID)

and

@esbot.event()
async def event_eventsub_notification_follow(payload: eventsub.ChannelFollowData) -> None:
    print('Received event!')
```
